src/Spout/Common/Escaper/XLSX.php

```diff
@@ -95,6 +95,10 @@ protected function getControlCharactersEscapingMap()
      */
     protected function escapeControlCharacters($string)
     {
+        if (1 !== preg_match('~[^A-Za-z0-9\-.:, ]~', $string)) {
```

Comment:
This does not work for characters that are not in the a-Z range: accented characters, different alphabets, etc..